src/Ombi/ClientApp/src/app/my-nav/nav-search.component.scss

```
@@ -1,75 +1,110 @@
-.option-img {
-    vertical-align: middle;
-    margin-right: 8px;
-  }
+.search-input {
+    flex: 1;
+    background: none;
+    border: none;
+    outline: none;
+    color: #fff;
+    font-size: 14px;
+    font-family: 'Inter', 'Roboto', sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)